FreeRTOS/FreeRTOS-Kernel
# Add runtime parameter checks

Description
-----------
This commit adds runtime checks for function parameters to mpu_wrappers_v2 file. The same checks are performed in the API implementation using asserts.

Test Steps
-----------
Ran the soak tests on STM32H7-NUCLEO board successfully.

Checklist:
----------




By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
